src/System.Linq.Expressions/src/System/Linq/Expressions/DynamicExpression.cs

```diff
@@ -453,25 +435,20 @@ public static new DynamicExpression MakeDynamic(Type delegateType, CallSiteBinde
         /// <returns>
         /// A <see cref="DynamicExpression" /> that has <see cref="NodeType" /> equal to
         /// <see cref="ExpressionType.Dynamic">Dynamic</see> and has the
-        /// <see cref="DynamicExpression.DelegateType">DelegateType</see>,
-        /// <see cref="DynamicExpression.Binder">Binder</see>, and
-        /// <see cref="DynamicExpression.Arguments">Arguments</see> set to the specified values.
+        /// <see cref="DelegateType">DelegateType</see>,
+        /// <see cref="Binder">Binder</see>, and
+        /// <see cref="Arguments">Arguments</see> set to the specified values.
         /// </returns>
         public static new DynamicExpression MakeDynamic(Type delegateType, CallSiteBinder binder, Expression arg0, Expression arg1, Expression arg2, Expression arg3)
         {
             return ExpressionExtension.MakeDynamic(delegateType, binder, arg0, arg1, arg2, arg3);
         }
+
         #endregion
 
-        Expression IDynamicExpression.Rewrite(Expression[] args)
-        {
-            return this.Rewrite(args);
-        }
+        Expression IDynamicExpression.Rewrite(Expression[] args) => Rewrite(args);
 
-        object IDynamicExpression.CreateCallSite()
-        {
-            return CallSite.Create(this.DelegateType, this.Binder);
-        }
+        object IDynamicExpression.CreateCallSite() => CallSite.Create(this.DelegateType, this.Binder);
```

Comment:
I think the new C# syntax is great for simple expressions, this one is IMO pushing it -- there is not huge difference in readability between the 2.